components/concept/engine/src/main/java/mozilla/components/concept/engine/Engine.kt

```diff
@@ -144,6 +145,24 @@ interface Engine {
     ): Unit = onError(UnsupportedOperationException("Clearing browsing data is not supported by this engine. " +
             "Check both the engine and engine session implementation."))
 
+    /**
+     * Fetch a list of trackers logged for a given [session] .
+     *
+     * @param session the session where the trackers were logged.
+     * @param onSuccess callback invoked if the data was fetched successfully.
+     * @param onError (optional) callback invoked if fetching the data caused an exception.
+     */
+    fun getTrackersLog(
+        session: EngineSession,
+        onSuccess: (List<TrackerLog>) -> Unit,
+        onError: (Throwable) -> Unit = { }
+    ): Unit = onError(
+        UnsupportedOperationException(
+            "getTrackersLog is not supported by this engine. " +
+                "Check both the engine and engine session implementation."
```

Comment:
`Check both` assumes there will only be two. :)

The first line should be good enough context.

Reply:
👍